anchore_engine/services/apiext/api/controllers/images.py

```diff
@@ -1570,3 +1573,42 @@ def list_secret_search_results(imageDigest):
         raise
     except Exception as err:
         raise api_exceptions.InternalError(str(err), detail={})
+
+
+@authorizer.requires([ActionBoundPermission(domain=RequestingAccountValue())])
+def get_image_sbom_native(imageDigest: str):
+    """
+    GET /images/{digest}/sboms/native
+
+    :param imageDigest:
+    :return: A native anchore (syft) formatted sbom for the image
+    """
+
+    account = ApiRequestContextProxy.namespace()
+    try:
+        _get_image_ok(account, imageDigest)
+
+        client = internal_client_for(CatalogClient, account)
+        resp = client.get_document("syft_sbom", imageDigest)
+
+        # Do
+        compressed_bytes = json_content_gzipper(resp)
+        resp = flask.Response(
```

Comment:
Can you set the name in the headers like so: https://github.com/anchore/enterprise/blob/960db726453d72b0043dba7bc48de8d2489d3fd6/anchore_enterprise/services/api/controllers/sources.py#L113

Reply:
Yes I can!

Reply:
I will also move this into the catalog api in a follow-up PR so that the API controller isn't hitting the object store directly.

Reply:
i'm doing the same thing in enterprise, so no complaints for now, haha. i'll make sure to make the same change